string name
int32 ID
int32 nRigidBodies
body[] bodies